libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -358,48 +383,6 @@ void ContractLevelChecker::annotateBaseConstructorArguments(
 
 }
 
-void ContractLevelChecker::checkConstructor(ContractDefinition const& _contract)
```

Comment:
This is completely removed without replacement? Is that intended?

Reply:
Nevermind, I found the replacement

Reply:
Just moved them, yeah.